src/Modules/ConfigOptions.lua

```diff
@@ -609,6 +609,7 @@ return {
 	end },
 	{ var = "buffLifetap", type = "check", label = "Do you have Lifetap?", ifCond = "Lifetap", apply = function(val, modList, enemyModList)
 		modList:NewMod("Condition:Lifetap", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
+		modList:NewMod("FlaskLifeRecovery", "INC", 20, "Config")
```

Comment:
We don't have a place to display sources for flask mods, but in the future if we do it'd be better to have this specify a source:
```suggestion
		modList:NewMod("FlaskLifeRecovery", "INC", 20, "Lifetap")
```